main.js

```diff
@@ -196,25 +196,41 @@ function loadImageWithPalette() {
 
 function hexToRgba(hex) {
   return [parseInt(hex.slice(1, 3), 16),
-    parseInt(hex.slice(3, 5), 16),
-    parseInt(hex.slice(5, 7), 16),
+  parseInt(hex.slice(3, 5), 16),
+  parseInt(hex.slice(5, 7), 16),
     255];
 }
 
```

Comment:
Can you add an extra comment separator here:

//
// Color distance helpers
//

Reply:
Done!